devenv/docker/blocks/prometheus/docker-compose.yaml

```diff
@@ -25,7 +25,11 @@
       FD_DATASOURCE: prom
 
   alertmanager:
-    image: quay.io/prometheus/alertmanager
+    image: prom/alertmanager 
+    volumes:
+    - ${PWD}/docker/blocks/prometheus/alertmanager.yml:/etc/alertmanager/alertmanager.yml
+    command: >
+      --config.file=/etc/alertmanager/alertmanager.yml
```

Comment:
technically not necessary as this is the default path defined in the Dockerfile for the image but I prefer explicit commands instead of having to dig around in the upstream dockerfiles